content/en/docs/concepts/services-networking/cluster-ip-allocation.md

```diff
@@ -0,0 +1,139 @@
+---
+reviewers:
+- sftim
+- thockin
+title: Service ClusterIP allocation
+content_type: concept
+weight: 120
+---
+
+
+<!-- overview -->
+
+In Kubernetes, [Services](/docs/concepts/services-networking/service/) are an abstract way to expose
+an application running on a set of Pods. Services
+can have a cluster-scoped virtual IP address (using a Service of `type: ClusterIP`).
+Clients can connect using that virtual IP address, and Kubernetes then load-balances traffic to that
+Service across the different backing Pods.
+
+<!-- body -->
+
+## How Service ClusterIPs are allocated?
+
+A Service `ClusterIP` can be assigned:
+
+_dynamically_
+: the cluster's control plane automatically picks a free IP address from within the configured IP range for `type: ClusterIP` Services.
+
+_statically_
+: you specify an IP address of your choice, from within the configured IP range for Services.
+
+Across your whole cluster, every Service `ClusterIP` must be unique.
+Trying to create a Service with a specific `ClusterIP` that has already
+been allocated will return an error.
+
```

Comment:
Should we mention what happens when you have a **dual stack** cluster and request a new Service with `type` set to `ClusterIP`?

Reply:
I prefer to keep this section to explain the mechanics of IP allocations to not confuse users, there is a section for DualStack already, the mechanics are the same for each family, but then we have different permutations Single, Dual, IPv4-IPv6, IPv6-IPv4 ... :/

Reply:
OK, if it's accurate then that's fine.